LAUNCH-SCRIPTS.md

```diff
@@ -15,7 +15,7 @@
 
 - `OT_CONDA_CMD`: Sets a custom Conda command or an absolute path to the binary (useful when it isn't in the user's `PATH`). If nothing is provided, we detect and use `CONDA_EXE` if available, which is a variable that's set by Conda itself and always points at the user's installed Conda binary.
 
-- `OT_CONDA_ENV`: Sets the name of the Conda environment. Defaults to `onetrainer`.
+- `OT_CONDA_ENV`: Sets the name (or an absolute/relative path) of the Conda environment. If a name or relative path is used, it will be relative to the OneTrainer directory. Defaults to `conda_env`.
```

Comment:
This one could be a bit more clear. As far as I understand, it's always interpreted as a path now. even if you pass "onetrainer", it will not use the system wide env called "onetrainer".

Reply:
That comment is modeled after the Venv option, since they both behave the same way now:

```
- `OT_PYTHON_VENV`: Sets the name (or an absolute/relative path) of the Python Venv. If a name or relative path is used, it will be relative to the OneTrainer directory. Defaults to `venv`.
```

The first sentence might be a bit confusing:

>  Sets the name (or an absolute/relative path) of the Conda environment.

But the second sentence sets that straight:

> If a name or relative path is used, it will be relative to the OneTrainer directory.

I can't think of another way to express this, which would be cleare.

Reply:
@Nerogar Hmm, actually. What if we just add one word?

- `OT_CONDA_ENV`: Sets the **directory** name (or an absolute/relative path) of the Conda environment. If a name or relative path is used, it will be relative to the OneTrainer directory. Defaults to `conda_env`.
- `OT_PYTHON_VENV`: Sets the **directory** name (or an absolute/relative path) of the Python Venv. If a name or relative path is used, it will be relative to the OneTrainer directory. Defaults to `venv`.

Reply:
Done via force-push for such a minor change:

https://github.com/Nerogar/OneTrainer/compare/d420e967bbf7f3eb4217b14c051c6a997429399f..1d3164afee30bba7457738090ff681eaf917eb9d

Reply:
that sounds good